packages/addons-v5/package.json

```diff
@@ -44,9 +45,8 @@
   "repository": "heroku/cli",
   "scripts": {
     "postpublish": "rm oclif.manifest.json",
-    "prepack": "oclif-dev manifest",
+    "prepack": "oclif manifest",
     "release": "np",
-    "test": "nyc mocha",
-    "version": "oclif-dev readme && git add README.md"
```

Comment:
I know we talked about this in Slack, but just pointing out that all of the README update functions seem to be deleted in this PR. That means the package READMEs won't get updated at all.

Reply:
I have no idea how this happened. I reverted the reverts and this was the outcome. I'll fix.

Reply:
Thank you.